Package_Pokemon.py

```diff
@@ -117,7 +117,7 @@
             move = copy(move_list[dump[i][0]])
             moves.append(move)
             if ability == 'Skill Link' and 'Hits 2-5 times' in move.effect:
-                move.power *= 5/2.1575
+                move.power *= 5/2.2575

```

Comment:
Looking again at the math on Bulbapedia (some of which is wrong), I think this line should be:
`move.power *= 3.167`

Reply:
indeed ! and the other place should be changed as well. i'll change that

Reply:
updated !

Reply:
Actually IDK why I even bothered adding the check for Skill Link. I don't think any of the rentals get it.

Reply:
indeed, found none there : https://www.serebii.net/swordshield/dynamaxadventurespokemon.shtml